src/main/kotlin/no/nav/familie/ks/sak/integrasjon/oppgave/domene/DbOppgave.kt

```diff
@@ -0,0 +1,42 @@
+package no.nav.familie.ks.sak.integrasjon.oppgave.domene
+
+import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
+import no.nav.familie.ks.sak.kjerne.behandling.Behandling
+import java.time.LocalDateTime
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.SequenceGenerator
+import javax.persistence.Table
+
+@Entity(name = "Oppgave")
+@Table(name = "OPPGAVE")
+data class Dboppgave(
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "oppgave_seq_generator")
+    @SequenceGenerator(name = "oppgave_seq_generator", sequenceName = "oppgave_seq", allocationSize = 50)
+    val id: Long = 0,
+
+    @ManyToOne
+    @JoinColumn(name = "fk_behandling_id", nullable = false, updatable = false)
+    val behandling: Behandling,
+
+    @Column(name = "gsak_id", nullable = false, updatable = false)
+    val gsakId: String,
```

Comment:
Hva står egentlig gsakId for? Stusser på g'n 😛

Reply:
det er en kobling til oppgaven i gsak/oppgave systemet

Reply:
Tror det står for gosys. Burde vi endre navnet til gosysId?

Reply:
Ja, hvis det står for gosys. 👍  Så fersk som jeg er i NAV så var det vanskelig å skjønne hva g står for.